Localize/lang/strings.json

```
@@ -776,17 +765,11 @@
   "ZnGRMP": "(UTC+07:00) Tomsk",
   "_++ZVe/.comment": "Title for testing section",
   "_+0yxlR.comment": "Label for the function display radio group",
-  "_+20fdv.comment": "Error message when having multiple invalid authentication properties",
-  "_+3efoo.comment": "Placeholder when no schema has been added",
   "_+DxBpE.comment": "Load schema error message",
-  "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
-  "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
+  "_+Uvo/p.comment": "Alt text for down chevron",
   "_+W9fkq.comment": "Timezone value ",
+  "_+ZSBrq.comment": "Accessibility label",
   "_+dlOgj.comment": "Timezone value ",
-  "_+g+mch.comment": "Label for description of custom multipartBody Function",
-  "_+iMTt7.comment": "aria label description for check button",
-  "_+jnjtp.comment": "Select a Manual Workflow resource",
-  "_+lpuMl.comment": "Required value parameter to return given if function is false",
   "_+xiT4Q.comment": "title for pagination setting",
   "_+ym4nO.comment": "Label for the Tracked Properties field in the settings section",
   "_//nQG4.comment": "Label for api selection",
```

Unsure to what API selection means, a connector operation?

```suggestion
  "_//nQG4.comment": "Label for API selection",
```

---

Yes